tests/Aspire.Hosting.PostgreSQL.Tests/PostgresFunctionalTests.cs

```diff
@@ -203,10 +203,8 @@ public async Task WithDataShouldPersistStateBetweenUsages(bool useVolume)
             var username = "postgres";
             var password = "p@ssw0rd1";
 
-            var usernameParameter = builder1.AddParameter("user");
-            var passwordParameter = builder1.AddParameter("pwd");
-            builder1.Configuration["Parameters:user"] = username;
-            builder1.Configuration["Parameters:pwd"] = password;
+            var usernameParameter = builder1.AddParameter("user", username);
```

Comment:
I like this new code, it is much simpler now that we have these overloads.

But can you help me understand what wasn't working before this change? Adding the Parameter and setting the value via the Configuration should just work.